### PR TITLE
Updates for Remote - SSH + Containers for 1.63 release

### PR DESCRIPTION
**Pull request by Chuxel**
I noticed we were missing docs on this feature in the Remote - Containers documentation. This resolves that and adds one item to the 1.63 remote release notes.

@gregvanl - I did this against main assuming we were pretty close to vnext being merged.  LMK if I should redo against vnext instead.

### CODE REVIEW

File: docs/remote/containers.md [hunk 1]
```diff
@@ -129,6 +129,18 @@ Once the WSL 2 engine is enabled, you can either:
...
 
+### Open a folder on a remote SSH host in a container
+
+As of VS Code 1.63, **you can use the [Remote - SSH](ssh.md) and Remote - Containers extensions together**! You do not even need to have a Docker client installed locally. To do so:
+
+1. Follow the [installation](/docs/remote/ssh.md#installation) and SSH [host setup](/docs/remote/ssh.md#ssh-host-setup) steps for the Remote - SSH extension.
+1. **[Optional]** Set up SSH [key based authentication](/docs/remote/troubleshooting.md#configuring-key-based-authentication) to the server so you do not need to enter your password multiple times.
+1. [Install Docker](#installation) on your SSH host. You do not need to install Docker locally.
+1. Follow the [quick start](/docs/remote/ssh.md#connect-to-a-remote-host) for the Remote - SSH extension to connect to a host and open a folder there.
+1. Use the **Remote-Containers: Reopen in Container.** command.
```
Comment (bamurtaugh) [inline docs/remote/containers.md:140]: Remove period from command? i.e. `**Remote-Containers: Reopen in Container**

File: remote-release-notes/v1_63.md [hunk 2]
```diff
@@ -37,7 +37,7 @@ A new syntax in the `devcontainer.json` features property allows for including c
 
 The feature is identified with a triplet `owner/repository/feature` and downloaded from that repository's releases as `features.tgz` with the feature being part of that tar file.
 
-This feature is currently in preview. We will provide detailed documentation on how to craft your own features as this effort proceeds.
+This feature is currently in preview. We will provide detailed documentation on how to craft your own features as this effort proceeds. [You can find an inital template with information on creating your own here](https://github.com/microsoft/dev-container-features-template).
```
Comment (bamurtaugh) [inline remote-release-notes/v1_63.md:40]: inital -> initial

File: remote/advancedcontainers/develop-remote-host.md [hunk 3]
```diff
@@ -5,15 +5,30 @@ TOCTitle: Develop on a remote Docker host
...
 
-## A basic remote example
+As of VS Code 1.63, **you can use the [Remote - SSH](ssh.md) and Remote - Containers extensions together**! You do not even need to have a Docker client installed locally. To do so:
+
+
+1. Follow the [installation](/docs/remote/ssh.md#installation) and SSH [host setup](/docs/remote/ssh.md#ssh-host-setup) steps for the Remote - SSH extension.
+1. **[Optional]** Set up SSH [key based authentication](/docs/remote/troubleshooting.md#configuring-key-based-authentication) to the server so you do not need to enter your password multiple times.
+1. [Install Docker](/docs/remote/containers#installation) on your SSH host. You do not need to install Docker locally.
+1. Follow the [quick start](/docs/remote/ssh.md#connect-to-a-remote-host) for the Remote - SSH extension to connect to a host and open a folder there.
+1. Use the **Remote-Containers: Reopen in Container.** command.
```
Comment (bamurtaugh) [inline remote/advancedcontainers/develop-remote-host.md:23]: Same as above; remove period from command?

**Comment by bamurtaugh** [comment]
Couple of small comments, otherwise looks fantastic! Thanks so much for adding this content @Chuxel

**Comment by gregvanl** [comment]
@Chuxel Yikes, big scary merge since you're updating v1_63.md in vnext and then docs/remote/containers.md in main.
I don't think we're releasing until tomorrow so it's probably better to target vnext so we don't accidently merge vnext to main before the actual release.

**Comment by Chuxel** [comment]
I can just resubmit to vnext - easy enough to do!  Will do that here in a bit and address Brigit's comments.

**Pull request closed without merging**